frontend/stryker.conf.json

```diff
@@ -12,8 +12,7 @@
         "projectType": "create-react-app"
     },
     "mutate": [
-        "src/main/**/*.{js,jsx,ts,tsx}",
-        "!src/main/**/*_NoStryker.{js,jsx,ts,tsx}"
+        "src/main/pages/HomePage.js"
```

Comment:
Wait, what?  This removes all mutation from every file except for the HomePage.js.

Is that your intention? 

I sort of think we ought to put this back the way it was.

Reply:
Hi, yes the mutation testing for frontend took a very long time so I just wanted to see for specifically the file I changed, but I will make the changes you suggested. Thank you for the feedback.